### DISCUSSION TITLE
Mergifyio defaults to use members of repository or organisation when no `update_bot_account` is given?

### DISCUSSION THREAD

**Discussion by Rakshith-R** [Mergifyio/mergify#2437] [Q&A]
hey, was bit confused whether mergifyio when no `update_bot_account` is given defaults to  using members of repository or organisation. 
> For certain actions, such as rebasing branches, Mergify has to impersonate a GitHub user. You can specify the account to use with this option. If no update_bot_account is set, Mergify picks randomly one of the organization users instead. The user account must have already been logged in Mergify dashboard once.

Is it not inconvenient using a user who is a member of the organisation but don't necessarily contribute to that particular repository for completing the action.
Thanks.

**Comment by jd** [comment]
I'm not sure there's an actual question here @Rakshith-R. If you have a suggestion/wish, feel free to open an issue though!

**Reply by Rakshith-R** [reply]
Does merigyio pick a member of repository like mentioned here https://docs.mergify.io/actions/merge/#strict-rebase or from entire organisation like in the post ? 
Ps: Mergifyio is 💯

**Reply by jd** [reply]
It takes it from the entire org and tries until one is allowed to do the operation, or end up reporting a failure.

**Reply by Rakshith-R** [reply]
The member it chooses needs to have write access to the repository for the operation to be successful?

**Reply by jd** [reply]
Yes

**Reply by Rakshith-R** [reply]
Thanks
